### DISCUSSION TITLE
Q: 使用自己服务器的数据库并在服务器独立部署也需要配置 Leancloud 吗？ || Q: Do I need to configure Leancloud when using my own server’s database and deploying it independently on the server?

### DISCUSSION THREAD

**Discussion by cn-ryh** [walinejs/waline#2463] [Q&A] [answered]
### Describe the bug | 问题描述

rt. 看到说明中 leadcloud 相关是必填项，但都用自己的了还需要 leadcloud 吗

### Website URL | 问题网站

https://waline.js.org/reference/server/env.html

### Where your waline deploy? | 服务部署在哪里？

Other

### Where your comment data store? | 数据存储在哪里？

MySQL

---
### Describe the bug | Problem description

rt. I saw that leadcloud is required in the description, but do I still need leadcloud if I use my own?

### Website URL | Problem website

https://waline.js.org/reference/server/env.html

### Where your waline deploy? | Where is the service deployed?

Other

### Where your comment data store? | Where is the data stored?

MySQL

**Comment by lizheming** [comment, marked as answer]
不需要，LeanCloud 不是必填的

---
No, LeanCloud is not required
